### ISSUE TITLE
Portfolio (Round 1) feedback

### ISSUE THREAD

**Issue by kjjohnsen** [csci48306830fa23/portfolio-leiliBV#1]
The videos are missing sound/voiceover/caption, description.  This is really important, so that I know what to focus on. 
Video 1 appears to illustrate keyboard control and audio.  It's a pretty good demo that showcases that you've learned how to do these things (Respond to events)
Video 2 shows a very similar thing, so It's not clear what it adds.
Video 3 is nice.  I like the effort put into this one to showcase the configurable joint.
It's unclear what's going on in video 4.  I'm going to guess it's about coroutines?
